zebra-scan/src/tests.rs

```diff
@@ -322,7 +276,7 @@ async fn scanning_fake_blocks_store_key_and_results() -> Result<()> {
     );
 
     // Scan with our key
-    let res = scan_block(
+    let res = zcash_client_backend::scanning::scan_block(
```

Comment:
Can we use the new function here?

Reply:
https://github.com/ZcashFoundation/zebra/pull/7994#discussion_r1408176439